presto-hive/src/main/java/com/facebook/presto/hive/BackgroundHiveSplitLoader.java

```diff
@@ -645,4 +665,27 @@ private static String getPartitionLocation(Table table, Optional<Partition> part
         }
         return partition.get().getStorage().getLocation();
     }
+
+    private static Optional<ColumnDomain<HiveColumnHandle>> getPathDomain(TupleDomain<HiveColumnHandle> effectivePredicate)
+    {
+        if (!effectivePredicate.getColumnDomains().isPresent()) {
+            return Optional.empty();
+        }
+
+        return effectivePredicate.getColumnDomains().get().stream()
+                .filter(columnDomain -> columnDomain.getColumn().getName().equalsIgnoreCase(PATH_COLUMN_NAME))
```

Comment:
Updated. Actually there seems to be a better way to check this: HiveColumnHandle.isPathColumnHandle. Fixed in next iteration,